x/blob/types/payforblob.go

```diff
@@ -186,15 +186,7 @@ func DefaultEstimateGas(blobSizes []uint32) uint64 {
 // tail padding).
 func ValidateBlobNamespace(ns appns.Namespace) error {
 	if ns.IsReserved() {
-		return ErrReservedNamespace.Wrapf("got namespace: %x, want: > %x", ns, appns.MaxReservedNamespace)
-	}
-
-	if ns.IsParityShares() {
-		return ErrParitySharesNamespace
```

Comment:
After integrating this PR, there will be no active usage for `ErrParitySharesNamespace`  in the app. I also initially thought of deleting it, however, there are other unused errors among the existing registered errors e.g., [ErrInvalidShareCommitments](https://github.com/celestiaorg/celestia-app/blob/84fa044b034e821610b8c7b93f827719432e609a/x/blob/types/errors.go#L20), so I decided to keep it as is. @rootulp Do you think there will be future usage for this error?

Reply:
On a related note, sometimes some errors are not used but kept so that their code number is not replaced by some other error which might break downstream for people depending on it

Reply:
I'm not too sure who uses the error code since the message is always accompanied with it in the response but I'd tend to agree that safest thing to do would be to reserve it (just by leaving a comment probably)